src/fileformats/ocd_georef_fields.cpp

```diff
@@ -0,0 +1,461 @@
+/*
+ *    Copyright 2013-2017 Kai Pastor
+ *    Copyright 2018 Libor Pecháček
+ *
+ *    This file is part of OpenOrienteering.
+ *
+ *    OpenOrienteering is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    OpenOrienteering is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with OpenOrienteering.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "ocd_georef_fields.h"
+
+#include <cmath>
+#include <vector>
+
+#include "core/crs_template.h"
+#include "fileformats/ocd_file_import.h"
+#include "fileformats/ocd_file_export.h"
+
+namespace OpenOrienteering {
+
+namespace {
+
+enum struct MapperCrs
+{
+	LOCAL,
+	UTM,
+	GAUSS_KRUEGER,
+	EPSG,
+};
+
+constexpr struct {
+	MapperCrs key;
+	const char* value;
+} crs_string_map[] {
+    { MapperCrs::LOCAL, "Local" },
+    { MapperCrs::UTM, "UTM" },
+    { MapperCrs::GAUSS_KRUEGER, "Gauss-Krueger, datum: Potsdam" },
+    { MapperCrs::EPSG, "EPSG" },
+};
+
+constexpr struct {
+	const char* key;
+	MapperCrs value;
+} crs_id_map[] {
+    { "Local", MapperCrs::LOCAL },
+    { "UTM", MapperCrs::UTM },
+    { "Gauss-Krueger, datum: Potsdam", MapperCrs::GAUSS_KRUEGER },
+    { "EPSG", MapperCrs::EPSG },
+};
+
+enum struct OcdGrid {
```

Comment:
Mind adding the no-all-caps rule to Mapper's Coding Style, along with examples? That will help me follow the rules more easily.

This is going against my habits. All caps is in my world an idiom for a constant. Doing that otherwise in OpenOrienteering requires a bit of effort from me.

Reply:
I see. But it is coding style in Qt, and it is even in the C++ Core guidelines. 

Note that you are labeled 'member', so you can add missing pieces to the Mapper wiki ;-).

Reply:
Just had a look at Qt style enums and can see it. I'll use that style.